server.go

```diff
@@ -414,6 +416,16 @@ func (s *Server) attachClient(cl *Client, listener string) error {
 	}
 
 	cl.ParseConnect(listener, pk)
+	if atomic.LoadInt64(&s.Info.ClientsConnected) >= s.Options.Capabilities.MaximumClients {
+		if cl.Properties.ProtocolVersion < 5 {
+			s.SendConnack(cl, packets.ErrServerUnavailable, false, nil)
+		} else {
+			s.SendConnack(cl, packets.ErrQuotaExceeded, false, nil)
```

Comment:
Maybe` ErrServerBusy` would be more appropriate, what do you think?

Reply:
Sure, I made the change to use ErrServerBusy instead.